Update docker install command for Rancher 2.5 (#2675)
rancher/docs: content/rancher/v2.x/en/installation/other-installation-methods/air-gap/install-rancher/_index.md

> **Note:** These installation instructions assume you are using Helm 3. For migration of installs started with Helm 2, refer to the official [Helm 2 to 3 migration docs.](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/) This [section]({{<baseurl>}}/rancher/v2.x/en/installation/options/air-gap-helm2) provides a copy of the older air gap installation instructions for Rancher installed on Kubernetes with Helm 2, and it is intended to be used if upgrading to Helm 3 is not feasible.

### Privileged Access for Rancher v2.5+

> When the Rancher server is deployed in the Docker container, a local Kubernetes cluster is installed within the container, then Rancher is deployed on the cluster. Because privileged access is required to run containers within containers, you will need to install Rancher with the `--privileged` option in order to access many features of Rancher, including deploying pods on the local Kubernetes cluster.

Reviewer: ```suggestion
When the Rancher server is deployed in the Docker container, a local Kubernetes cluster is installed within the container for Rancher to use. Because many features of Rancher run as deployments, and privileged mode is required to run containers within containers, you will need to install Rancher with the `--privileged` option.
```

Feel free to refactor the grammar, but the previous way suggested that it was possible to run rancher without privileged and some features just wouldn't run, but Rancher won't run at all without the privileged flag now.
